wsproto/connection.py

```diff
@@ -186,6 +160,13 @@ def send_data(self, payload, final=True):
         self._outgoing += self._proto.send_data(payload, final)
 
     def close(self, code=CloseReason.NORMAL_CLOSURE, reason=None):
+        """
+        Initiate the close handshake by sending a CLOSE control message.
+
+        A clean teardown requires a CLOSE control messages from the other
+        endpoint before the underlying TCP connection can be closed, see
+        :class:`ConnectionClosed <wsproto.events.ConnectionClosed>`.
```

Comment:
You can write this more compactly as
```
:class:`~wsproto.events.ConnectionClosed`
```
([see](http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-syntax))

Reply:
👍